pkg/daemon/ceph/client/command.go

```diff
@@ -88,6 +88,11 @@ func ExecuteCephCommand(context *clusterd.Context, clusterName string, args []st
 	return executeCephCommandWithOutputFile(context, clusterName, false, args)
 }
 
+// ExecuteCephCommandDebug executes the 'ceph' command with debug output
+func ExecuteCephCommandDebug(context *clusterd.Context, clusterName string, debug bool, args []string) ([]byte, error) {
```

Comment:
My interpretation of this is that this is intended to execute the ceph command with debug output always turned on. Can `debug bool` be removed from the parameters and `true` be passed to `executeCephCommandWithOutputFile`?

Reply:
There is a caller that wants debug mode and another that might not want debug mode, depending on a `debug` flag it is passed to get the status. I'd be inclined to leave it with this name unless you have a better name in mind.

Reply:
This file kind of smells to me. There are so many seemingly arbitrary permutations of execute command. I wonder why not add the `debug bool` parameter to all the existing ones instead of making a new function.

I think this has kind of exploded over time to the point where there should be a proper design for these where there is one execute function that can take all of the options that have been turned into different permutations of function calls.

Reply:
For a long time I've also wants to refactor these helpers and the `exec` package for running processes. I'd like to have a struct containing properties that determines things like debug settings so we don't have to add a new param every time we need to configure it slightly differently. But this would be a much bigger change for another day, thus, smaller changes like this that are messy over time...